### DISCUSSION TITLE
TXT File when Multiple MRF are on the Same Page

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#143] [Q&A] [answered]
My web page has multiple MRF listed on the same page for multiple locations. Can I get guidance on how this should be setup? I received a violation letter saying it was out of compliance but I tested the cms-hpt.txt and my website opens. Do I complete the TXT File Generator for each MRF and save multiple TXT files into my root folder? Also should it open to the main website page or the page where the files are found.

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question. 

Since your hospital is under compliance review, please send your inquiry to [HPTCompliance@cms.hhs.gov](mailto:HPTCompliance@cms.hhs.gov).
